interfaces/systemd/backend.go

```diff
@@ -78,18 +77,16 @@ func (b *Backend) Setup(snapInfo *snap.Info, devMode bool, repo *interfaces.Repo
 			logger.Noticef("cannot reload systemd state: %s", err)
 		}
 	}
-	// Start any new services
+	// Start and enable any new services
 	for _, service := range changed {
-		err := systemd.Start(service)
-		if err != nil {
-			logger.Noticef("cannot start service %q: %s", service, err)
+		if err := systemd.EnableNow(service); err != nil {
```

Comment:
We need to be carefully here. If we restart the service files generated from the GPIO interface we actually remove the /sys/class/gpio/gpip<n>/ path which an application might use currently. Will see what we can do here.